fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserFragmentStore.kt

```diff
@@ -7,21 +7,27 @@
 package org.mozilla.fenix.components.toolbar
 
 import mozilla.components.lib.state.Action
+import mozilla.components.lib.state.Middleware
 import mozilla.components.lib.state.State
 import mozilla.components.lib.state.Store
 
 // The state that used to live in this class was moved into another component in #4281. Keeping
 // the shell of this file because we will need to expand it as we add additional features to
 // the browser.
-class BrowserFragmentStore(initialState: BrowserFragmentState) :
+class BrowserFragmentStore(
+    initialState: BrowserFragmentState,
+    middleware: List<Middleware<BrowserFragmentState, BrowserFragmentAction>>
+) :
     Store<BrowserFragmentState, BrowserFragmentAction>(initialState, ::browserStateReducer)
 
 /**
  * The state for the Browser Screen
  */
 class BrowserFragmentState : State
```

Comment:
This whole file is kinda wild that it still exists. At the top, it says it is unused and it doesn't even hold any state. This might be better renamed `BrowserToolbarStore` to better serve your purpose. 

Alternatively, I am also wondering if we should react to the `InitAction` from `BrowserAction` for your toolbar CFR middleware.